audio/libmad/Portfile

```diff
@@ -33,13 +33,21 @@ post-extract {
     touch ${worksrcpath}/AUTHORS ${worksrcpath}/ChangeLog ${worksrcpath}/NEWS
 }
 
-patchfiles      patch-configure.ac.diff automake.patch
+patchfiles      patch-configure.ac.diff automake.patch remove-optimizations.patch
 
 post-patch {
     reinplace "s|%PREFIX%|${prefix}|g" ${worksrcpath}/mad.pc
     reinplace "s|%VERSION%|${version}|g" ${worksrcpath}/mad.pc
 }
 
+pre-configure {
+    system -W ${worksrcpath} "aclocal --force"
+    system -W ${worksrcpath} "glibtoolize --force"
+    system -W ${worksrcpath} "autoheader --force"
+    system -W ${worksrcpath} "automake --force --add-missing"
+    system -W ${worksrcpath} "autoconf --force"
+}
```

Comment:
Usually we don't do this manually; we do `use_autoreconf  yes`. And this port already does this. This isn't enough?

Reply:
We need to regenerate the entire build system as it seems that these were generated with such an old version that `-apple-darwin20.10` is not a valid flag. This forces a rebuild because `use_autoreconf yes` did not appear to do it.

Reply:
Let's try to find out why `use_autoreconf yes` didn't work, because that's what we want to use. In many ports, we add `autoreconf.args -fvi`: does that help?